sklearn/naive_bayes.py

```diff
@@ -379,8 +414,11 @@ def _partial_fit(self, X, y, classes=None, _refit=False,
             self.sigma_[i, :] = new_sigma
             self.class_count_[i] += N_i
 
+        class_prior = self.class_prior
+
         self.sigma_[:, :] += epsilon
-        self.class_prior_[:] = self.class_count_ / np.sum(self.class_count_)
+        self._update_class_prior(class_prior=class_prior)
```

Comment:
Do you think it would be a good idea to update only if `fit_prior` is True and `self.class_prior` is not provided and set `self.class_prior_` at the first call to `_check_partial_fit_first_call`

It would prevent recreating arrays of shape `(n_classes,)` for every partial_fit call and repeated checks.